python/lsst/ts/dsm/dsm_csc.py

```diff
@@ -30,9 +30,13 @@ class DSMCSC(salobj.BaseCsc):
 
     enable_cmdline_state = True
     valid_simulation_modes = (0, 1, 2)
+    version = __version__
```

Comment:
Please consider adding a note to the class doc string or a code comment here (just above line 32) that the simulation modes are documented in the user guide. (I looked for them in the class doc string first, and would have appreciated the pointer).

Reply:
Done.